src/main/java/pt/tqs/g205/resources/CategoriaPratoResource.java

```diff
@@ -0,0 +1,30 @@
+package pt.tqs.g205.resources;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import pt.tqs.g205.domain.CategoriaPrato;
+import pt.tqs.g205.services.CategoriaPratoService;
+
+/**
+ * 
+ * 
+ *
+ */
+@RestController
+@RequestMapping(value = "/api/categorias")
+public class CategoriaPratoResource {
+
+  @Autowired
+  private CategoriaPratoService categoriaService;
+
+  @RequestMapping(value = "/pratos", method = RequestMethod.GET)
+  public ResponseEntity<?> getAll() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=pt.tqs.g205%3Arestapi&pullRequest=5&issues=AWNCwXmIXOBV_Yrh-xNu&open=AWNCwXmIXOBV_Yrh-xNu)

[ISSUE_KEY]: AWNCwXmIXOBV_Yrh-xNu